src/main/java/jeryl/fyp/model/student/ProjectStatus.java

```diff
@@ -0,0 +1,56 @@
+package jeryl.fyp.model.student;
+
+import static java.util.Objects.requireNonNull;
+import static jeryl.fyp.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's project status in the FYP manager.
+ * Guarantees: immutable; is valid as declared in {@link #isValidProjectStatus(String)}
+ */
+public class ProjectStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Status should only be of the form YTS, IP OR DONE";
+
+    public final String projectStatus;
+
+    //enum constants
+    enum Status {
```

Comment:
This enum seems to be unused in the end, how about removing it instead?

Reply:
Will remove it as you suggested!